### PR TITLE
Fix JENKINS-28161

### PR DESCRIPTION
**Pull request by rsandell**
This just fixes the NPE symptoms, but I'm afraid that
the reason that an event has a null value has some deeper issue somewhere.

### CODE REVIEW

**Comment by rsandell** [comment]
@reviewbybees

**Comment by jglick** [comment]
BTW please include a meaningful summary of the fix in the PR, and link to the JIRA issue from the description.

File: src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/model/BuildMemory.java [hunk 1]
```diff
@@ -60,11 +60,19 @@
...
+            if (o1 == null && o2 == null) {
+                return 0;
+            }
+            if (o1 != null && o2 == null) {
+                return 1;
+            }
+            if (o1 == null && o2 != null) {
+                return -1;
+            }
+            return new Integer(o1.hashCode()).compareTo(o2.hashCode());
```
Comment (stephenc) [inline src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/gerritnotifier/model/BuildMemory.java]: I know this is an aside from the current issue, but comparing events based on a 32 bit integer in order to 

> This ensures that every event received from Gerrit is kept track of individually

Seems a bit naïve

**Comment by stephenc** [comment]
:+1: on making more stable

**Pull request merged**